app/src/main/java/com/ezlevup/ganbyeong24/presentation/components/DatePickerField.kt

```
@@ -0,0 +1,159 @@
+                fontWeight = FontWeight.Bold,
+                color = MaterialTheme.colorScheme.onSurface,
+                modifier = Modifier.padding(bottom = 8.dp)
+        )
+
+        OutlinedTextField(
+                value =
+                        if (selectedDate.isNotEmpty()) {
+                            try {
+                                val date = dateFormatter.parse(selectedDate)
+                                displayFormatter.format(date ?: Date())
+                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)